client/src/lib/dependency-injection/tests/SchemaStateManager-test.js

```diff
@@ -0,0 +1,92 @@
+/* global jest, jasmine, describe, beforeEach, it, pit, expect, process */
+import SchemaStateManager from '../SchemaStateManager';
+
+describe('SchemaStateManager', () => {
+  let manager = null;
+  const state = {
```

Comment:
build state in the `beforeEach()` call, so any mutations that could possibly happen in unit tests are shielded from other unit tests